src/pages/ProfilesExplorerView/domain/variables/FiltersVariable/FiltersVariable.tsx

```diff
@@ -20,31 +20,35 @@ export class FiltersVariable extends AdHocFiltersVariable {
       name: key,
       label: 'Filters',
       filters: FiltersVariable.DEFAULT_VALUE,
+      expressionBuilder: (filters) =>
```

Comment:
We do this to prevent extra escaping of values with regex expressions (which happens with the `in` operator, which converts internally to the `=~` Prometheus label selector).